src/providers/logs/logsHelper.ts

```diff
@@ -0,0 +1,39 @@
+import { LogType, Log } from '../../shared/models/log.model';
```

Comment:
We should mock this and use the mock in the tests as it uses plugins which are not available on non ios devices.

Happy to chat about this one in the morning